src/EventTickets/resources/admin/components/EventDetailsPage/TicketTypesSection/index.tsx

```diff
@@ -1,13 +1,65 @@
-import {__} from '@wordpress/i18n';
+import {__, sprintf} from '@wordpress/i18n';
+import styles from './TicketTypesSection.module.scss';
+import SectionTable from '../SectionTable';
+import {useState} from 'react';
+
+const amountFormatter = new Intl.NumberFormat(navigator.language || navigator.languages[0], {
+    style: 'currency',
+    currency: window.GiveEventTicketsDetails.currencyCode,
+});
 
 /**
+ * Displays a blank slate for the EventTickets table.
+ *
  * @unreleased
  */
+const BlankSlate = () => {
+    const imagePath = `${window.GiveEventTicketsDetails.pluginUrl}/assets/dist/images/list-table/blank-slate-event-tickets-icon.svg`;
+    return (
+        <div className={styles.container}>
+            <img src={imagePath} alt={__('No ticket created yet', 'give')} />
+            <h3>{__('No ticket created yet', 'give')}</h3>
+            <p className={styles.helpMessage}>{__('Create a ticket to complete your event setup.', 'give')}</p>
+            <p>
+                {/*Todo: Set an onClick event to open the CreateTicketTypeModal*/}
+                <a
+                    href={`${window.GiveEventTicketsDetails.adminUrl}edit.php?post_type=give_forms&page=give-event-tickets&new=event`}
```

Comment:
@pauloiankoski when I click "Create ticket" it redirects to the Events List Table with the Create Event modal open.

Reply:
Disregard.